src/arch/helperadvsimd.h

```diff
@@ -43,6 +43,14 @@ typedef int32x2_t vint;
 static INLINE int vavailability_i(int name) { return 3; }
 static INLINE void vprefetch_v_p(const void *ptr) { }
 
+static INLINE int vtestallones_i_vo32(vopmask g) {
+  return vget_lane_u32(vpmin_u32(vand_u32(vget_low_u32(g), vget_high_u32(g)), vand_u32(vget_low_u32(g), vget_high_u32(g))), 0);
```

Comment:
Your code does not compile. vpminq_u32 takes two arguments.

vtestallones_i_vo64 and vtestallones_i_vo32 are basically same as _mm_test_all_ones in SSE4.1, but we can assume each int32_t lane is either 0 or -1.